portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -14,9 +14,7 @@
 
 package com.google.sps.data;
 
-/** An item on a todo list. */
 public final class Comment {
```

Comment:
I'd still add some comments for the class. Although it may seem redundant, in reality, many code search or editor will have the function that shows the comments when you hover over the class name. That's when the comment is especially useful.

Reply:
Added a comment!